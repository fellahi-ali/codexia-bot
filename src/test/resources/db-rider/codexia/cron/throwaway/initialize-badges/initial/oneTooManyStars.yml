codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 123
        coordinates: "test-project1/test-repo1"
        author: "some-first-author2"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

codexia_review:
    -
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 5
        author: TOO_MANY_STARS
        reason: 12430
        text: "The repo gained too many stars: 12430."

codexia_badge: []
