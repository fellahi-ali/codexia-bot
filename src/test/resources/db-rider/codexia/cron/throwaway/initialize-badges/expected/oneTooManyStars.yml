codexia_badge:
    -
        codexia_project_id: 5
        badge: "bad"
        deleted_at: null
